Packages/ContractusAPI/Sources/ContractusAPI/Models/Deal.swift

```diff
@@ -167,6 +167,23 @@ public struct Deal: Decodable {
         }
         return nil
     }
+    
+    public var timeFromStartFormatted: String {
```

Comment:
https://developer.apple.com/documentation/foundation/relativedatetimeformatter

https://www.hackingwithswift.com/example-code/system/how-to-show-a-relative-date-and-time-using-relativedatetimeformatter